src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerEngine.cs

```diff
@@ -65,17 +68,34 @@ public static AnonymizerEngine CreateWithFileContext(string configFilePath, stri
         public ITypedElement AnonymizeElement(ITypedElement element, AnonymizerSettings settings = null)
         {
             EnsureArg.IsNotNull(element, nameof(element));
+            try
+            {
+                ElementNode resourceNode = ElementNode.FromElement(element);
+                return resourceNode.Anonymize(_rules, _processors);
+            }
+            catch(Exception ex)
+            {
+                if(!(ex is AnonymizerConfigurationErrorsException) && _configurationManager.Configuration.processingErrors == ProcessingErrorsOption.skip)
```

Comment:
We can catch AnonymizerConfigurationErrorsException separately to make it more clear.

Reply:
Fixed it, thanks!